### PR TITLE
:seedling: Code cleanup

### PR DESCRIPTION
**Pull request by NotTheEvilOne**
**What this PR does / why we need it**:

This PR cleans up most basic scaffold traces in the code.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -21,11 +21,12 @@ import (
...
-	infrastructureclusterstackxk8siov1alpha1 "github.com/sovereignCloudStack/cluster-stack-provider-openstack/api/v1alpha1"
+	apiv1alpha1 "github.com/sovereignCloudStack/cluster-stack-provider-openstack/api/v1alpha1"
 	"github.com/sovereignCloudStack/cluster-stack-provider-openstack/internal/controller"
 	"k8s.io/apimachinery/pkg/runtime"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	runtimeutils "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
```
Comment (chess-knight) [inline cmd/main.go:30]: Linter seems doesn't like empty line here:
```suggestion
	runtimeutils "k8s.io/apimachinery/pkg/util/runtime"
	"k8s.io/client-go/kubernetes/scheme"
	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
```

Comment (NotTheEvilOne) [reply cmd/main.go]: It's automatically reformatted by the IDE that way. Nevertheless it's changed now to prevent it :)

**Review by matofeder — CHANGES_REQUESTED** [review]
Hi @NotTheEvilOne 
I see this PR still as a valid and useful one, just please resolve conflicts and rebase on the latest main. Then we can review it. Thx

**Comment by NotTheEvilOne** [comment]
Done. It's not much that's left thought :)

**Review by matofeder — APPROVED** [review]
lgtm

**Pull request merged**